tools/ResourceUpdater/main.cpp

```diff
@@ -829,6 +836,13 @@ bool update_battle_chars_info(const std::filesystem::path& input_dir, const std:
             }
         }
 
+        if (auto chars_opt = battle_data_json.find<json::object>("chars")) {
+            auto& chars_id = chars_opt.value();
+            if (auto char_opt = chars_id.find(id)) {
+                char_new_data["name_tw"] = char_opt->at("name_tw");
+            }
+        }
```

Comment:
try `char_new_data["name_tw"] = battle_data_json.get("chars", id, cn_default_name);`

Reply:
Is it that easy and compact? XD

Reply:
Ok this is game changing. I can implement `.get` in the operator localization that I implemented a week ago. tyvm